website/docs/downloads/windows.md

```diff
@@ -25,6 +33,28 @@ the changes and improvements introduced by this release.
     If Windows 8.x or Windows 10 prevents you from running DOSBox Staging via
     Microsoft Defender SmartScreen, see the [section below](#microsoft-defender-smartscreen).
 
+## Windows 7
+
+For 64-bit Windows 7, use the 64-bit MSVC build. It can be optionally selected
+in the [installer](0_80_1_x64_INSTALLER) and is also available as a [portable
+ZIP archive](0_80_1_x64_ZIP).
+
+For 32-bit Windows 7, use the 32-bit MSVC build [portable ZIP archive](0_80_1_x32_ZIP)
+
+## Windows Vista
+
+The official distribution packages only support Windows Vista up to version
+0.78.1.
+
+More recent releases can be built from the source by excluding FluidSynth and
+Slirp support. Please see the [Windows MSYS2 build instructions][win-build]
+for further details.
+
+[win-build]: https://github.com/dosbox-staging/dosbox-staging/blob/main/docs/build-windows.md#build-using-msys2
+
+## Harware requirements
+
+SSE 4.2 instruction set is required for version 0.75.1 or newer.
```

Comment:
@Torinde Does this apply to the macOS and Linux builds as well? I'm guessing it would be much less of a problem on macOS as the hardware and supperted macOS version are closely coupled, but Linux users can have pretty much any CPU?

Reply:
from [0.75.1 release notes](https://dosbox-staging.github.io/downloads/release-notes/0.75.1/#provide-windows-x64-builds):
> Both Windows builds now require SSE 4.2 instruction set, which is now on par with the Linux and macOS builds that have required SSE 4.2 support since the 0.75.0 release.

I was thinking on mentioning that on the About page (as example of what "limited CPU/memory" is unsupported), but it seems too much for _there_ ("e.g. CPUs without SSE4.2 are not supported in all macOS, Linux builds and in Windows build 0.75.1 or newer")

If you think it's relevant - I can add "Hardware requirements" chapters to the Linux and macOS download pages and mention it as hard requirement (e.g. for ALL Staging versions).

Reply:
...and my _interpretation_ of the above note is that macOS/Linux "0.75.0-RC1" and "0.75.0 pre-release" also require SSE4.2.

Reply:
> the hardware and supported macOS versions are closely coupled

There are a few Intel macs without SSE4.2 - there is even an unofficial patch: "[MouSSE is a partial SSE4.2 emulator that allows the MacPro3,1 to use the newer AMD drivers.](https://julianfairfax.gitlab.io/documentation/patching-macos-to-run-on-unsupported-macs)"

Interesting if Staging will work with that patch... but I can't test that.

Reply:
OK, so I'll add a note to Linux/macOS pages later today...

Reply:
Done.